docs/release-notes-v14.0.md

```diff
@@ -0,0 +1,84 @@
+    * Percona Blog: [PostgreSQL 14 – Performance, Security, Usability, and Observability](https://www.percona.com/blog/postgresql-14-performance-security-usability-and-observability/)
+
+The following is the list of extensions available in Percona Distribution for PostgreSQL.
+
+| Extension           | Version        | Description                  |
+| ------------------- | -------------- | ---------------------------- |
+| [Patroni](https://patroni.readthedocs.io/en/latest/) | 2.1.0 | a HA (High Availability) solution for PostgreSQL |
+| [Pgaudit](https://www.pgaudit.org/)             | 1.6.0 Beta | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+|[`pgAudit set user`](https://github.com/pgaudit/set_user)| 3.0.0|  provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.34    | a backup and restore solution for PostgreSQL       |
+|[`pgBadger`](https://github.com/darold/pgbadger) | 11.5       | a fast PostgreSQL Log Analyzer.|
+|[`pgBouncer`](https://www.pgbouncer.org/) | 1.16.0 | lightweight connection pooler for PostgreSQL|
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.7   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)| 0.9.2 - Beta1 | collects and aggregates statistics for PostgreSQL and provides histogram information.       |
+| [PostgreSQL Common](https://packages.debian.org/sid/percona-postgresql-common)| 226 | PostgreSQL database-cluster manager. It provides a structure under which multiple versions of PostgreSQL may be installed and/or multiple clusters maintained at one time.|
+|[`wal2json`](https://github.com/eulerto/wal2json) |2.3        | a PostgreSQL logical decoding JSON output plugin.|
+
+Percona Distribution for PostgreSQL also includes the following packages:
+
+* `llvm` 11.0.0 packages for Red Hat Enterprise Linux 8 / CentOS 8. This fixes compatibility issues with LLVM from upstream.
+* supplemental `ETCD` packages which can be used for setting up Patroni clusters. These packages are available for the following operating systems:
+
+|  Operating System   | Package              | Version | Description        |
+| ------------------- | ---------------------| --------| ------------------ |
+| CentOS 7            |`python3-python-etcd` | 0.4.3   | A Python client for ETCD     |
+| CentOS 8            | `etcd`               | 3.3.11  | A consistent, distributed key-value store|
+|                     | `python3-python-etcd`| 0.4.3   | A Python client for ETCD     |
+| Debian 9 ('stretch')| `etcd`               | 3.3.11  | A consistent, distributed key-value store|
+|                     | `python3-etcd`       | 0.4.5   | A Python client for ETCD     |
```

Comment:
@EvgeniyPatlan can you please review and double check these version numbers? Thank you!